Jellyfin.Data/Entities/MediaSegment.cs

```diff
@@ -0,0 +1,35 @@
+using System;
+using Jellyfin.Data.Enums;
+
+namespace Jellyfin.Data.Entities;
+
+/// <summary>
+///     An entity representing the metadata for a group of trickplay tiles.
+/// </summary>
+public class MediaSegment
+{
+    /// <summary>
+    ///     Gets or sets the id of the media segment.
+    /// </summary>
+    public Guid Id { get; set; }
+
+    /// <summary>
+    ///     Gets or sets the id of the associated item.
+    /// </summary>
+    public Guid ItemId { get; set; }
```

Comment:
To make it more obvious we might want to rename this to `MediaSourceId`, wdyt?

Reply:
i named that in line with other entities "itemid" like trickplay. I would keep it in line with that or refactor the other existing ones too.

Reply:
IMHO `ItemId` migth be misleading. Is trickplay even media source based (if not, this should be changed...)?